src/views/activity/activity_index_view.js

```diff
@@ -14,11 +14,15 @@ export default BaseView.extend({
   },
 
   afterRender () {
+    const startIndex = 1;
+
     this.removeSubviews();
-    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, 1, 3);
+    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, startIndex, 3);
     this.renderCollection(items, ActivityItemView, '.activity-results');
 
     // if there are results then show otherwise hide
-    this.adapter.results.length ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
+    this.adapter.results.length > startIndex ? this.show() : this.hide(); // eslint-disable-line no-unused-expressions
```

Comment:
Does this mean we only show activity items if there are 2 or more? If so, let's update the code comment just above this line to keep it truthful.